library/src/main/java/com/bumptech/glide/module/AppliesOptions.java

```diff
@@ -4,7 +4,7 @@
 import com.bumptech.glide.GlideBuilder;
 
 /**
- * An internal interface, to be removed when {@link GlideModule}s are removed.
```

Comment:
This was correct before, we want to refer to the deprecated GlideModule class.

Reply:
Oops, SGTM.